index.html

```diff
@@ -9,7 +9,12 @@
 </head>
 <body>
   <header>
+<<<<<<< HEAD
+    <h1><a href="" rel="home"> Dominator Flight Booking</a></h1>
+      <!-- nav ?-->
+=======
```

Comment:
Remove your merge lines like <<<<<HEAD and ====== from all of your files that have them. index.html has them at 12, 15, 17, 84, 88 and 91